core/execution/future/special_orders.go

```diff
@@ -123,13 +122,11 @@ func (m *Market) repriceAllSpecialOrders(
 	ctx context.Context,
 	changes uint8,
 	orderUpdates []*types.Order,
```

Comment:
orderUpdates can be removed here. we also probably don't need to return anything anymore

Reply:
orderUpdates are for the same thing as before. For the updates - don't we want to change the prices again after the resubmitting of the pegged orders?